clang/include/clang/StaticAnalyzer/Core/PathSensitive/EntryPointStats.h

```diff
@@ -0,0 +1,162 @@
+// EntryPointStats.h - Tracking statistics per  entry point -*- C++ -*-//
```

Comment:
```suggestion
// EntryPointStats.h - Tracking statistics per entry point --*- C++ -*-//
```
Just a doubled space.